src/bundle/Resources/translations/validators.en.xliff

```diff
@@ -76,6 +76,11 @@
         <target state="new">Selected Location has no children.</target>
         <note>key: ezplatform.trash.location_has_no_children</note>
       </trans-unit>
+      <trans-unit id="ea852842b560d609441b8470b79735f2dc91698f" resname="ibexa.language.language_code.format">
+        <source>The Language code {{ value }} contains illegal characters. Language code should start with a letter, digit or underscore and only contain letters, digits, numbers, underscores ("_"), hyphens ("-") and colons (":").").</source>
+        <target state="new">The Language code {{ value }} contains illegal characters. Language code should start with a letter, digit or underscore and only contain letters, digits, numbers, underscores ("_"), hyphens ("-") and colons (":").").</target>
+        <note>key: ibexa.language.language_code.format</note>
+      </trans-unit>
```

Comment:
All translation messages must be extracted by running the command
```php bin/console translation:extract --config=ibexa_admin_ui``` 
The way to do it is to find or create a class that implements 
```JMS\TranslationBundle\Translation\TranslationContainerInterface```
and put there proper 
```JMS\TranslationBundle\Model\Message```